tests/HTMLTest.php

```diff
@@ -114,6 +114,20 @@ public function testHeaderTable() : void
         self::assertNotContains('==', $document);
     }
 
+    /**
+     * Testing HTML table with headers
+     */
+    public function testTableWithNestedList() : void
+    {
+        $document = $this->parseHTML('table-nested-list.rst');
+
+        self::assertSame(1, substr_count($document, '<ul>'));
```

Comment:
Would it make sense to use Phpunit's DOM additions to check this?

Reply:
i don't know this plugin, but as you can see, there are a lot assertions about the DOM created by the parser in this file.
Maybe if it makes things easier, it would make sens

Reply:
Maybe create an issue to improve and standardize how we are checking the dom in tests and we can make that change in another PR?